README.md

```diff
@@ -1,5 +1,9 @@
 # Adobe Experience Platform Roku SDK
 
+[![release version](https://img.shields.io/github/v/release/adobe/aepsdk-roku.svg?label=AEP%20Roku%20SDK%20Release&logo=github)](https://github.com/adobe/aepsdk-roku/releases)
+[![build Roku](https://github.com/adobe/aepsdk-roku/actions/workflows/build.yml/badge.svg)](https://github.com/adobe/aepsdk-roku/actions/workflows/build.yml)
```

Comment:
nit: label: Replace `build Roku SDK` -> `build` 
This is coming from Action/workflow so we will need to update the workflow name.